__tests__/server.test.js

```diff
@@ -73,6 +73,7 @@ describe('GET /api/articles/:article_id', () => {
                 votes: 100,
                 article_img_url:
                   "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+                comment_count: "11"
```

Comment:
Ideally we should not be going back and altering previous tests, but it is necessary here because of the jest matcher you have used. toEqual would now fail with the comment count included. If you had a different matcher that does not need the whole object to match (toMatchObject maybe) then your previous test would still pass and you could write a separate test for this new ticket. toEqual needs all of the properties to be in there and to match. You don't have to change it, but just wanted to give you some clarification on the matchers and the flow of testing!